misk-zookeeper/build.gradle

```diff
@@ -9,13 +9,20 @@ apply plugin: 'kotlin-jpa'
 dependencies {
   compile(dep.curatorFramework) {
     exclude(group: 'org.slf4j', module: 'slf4j-log4j12')
+    exclude(group: 'log4j', module: 'log4j')
   }
   compile project(':misk')
-  compile dep.zookeeper
+  compile(dep.zookeeper) {
+    exclude(group: 'org.slf4j', module: 'slf4j-log4j12')
+    exclude(group: 'log4j', module: 'log4j')
+  }
 
   testCompile dep.docker
   testCompile dep.junitApi
-  testCompile dep.zookeeper
+  testCompile(dep.zookeeper) {
```

Comment:
`testCompile` includes the configurations from `compile` does it not?

Reply:
the excludes don't propagate

Reply:
so, I'm still not totally sure this'll work, but hopeful